ExtLibs/Xamarin/Xamarin/Resources/mavcmd.xml

```diff
@@ -210,7 +210,16 @@
       <Y></Y>
       <Z></Z>
     </DO_SET_CAM_TRIGG_DIST>
-    <DO_SET_RELAY>
+    <DO_SET_HOME>
+      <P1>Current(1)/Spec(0)</P1>
```

Comment:
this is fine of course but I might shorten it to just "Current(1)".  just an idea though, I'm fine either way

Reply:
was just a copy and paste...not sure I know exactly how to change and if we do, shouldn't we change the other instances here?